src/bitcoind.cpp

```diff
@@ -54,22 +54,19 @@ static bool AppInit(int argc, char* argv[])
     SetupServerArgs(node);
```

Comment:
In commit "scripted-diff: gArgs -> args" (fa9d5902f7d72e8cce105dd1b1f5a1062e304b10)

Is the second line of the scripted diff actually doing anything?

```
sed -i 's/&args;/\&gArgs;/g' src/init.cpp
```

I don't see a `&gArgs`

Reply:
It is undoing one false replacement of the first line in the script. Offending replacement would be:

```diff
-node.args = &gArgs;
+node.args = &args;